### PR TITLE
Add a hook to merge public and party contract streams for Daml Hub

### PR DESCRIPTION
**Pull request by alexmatson-da**

### CODE REVIEW

File: ui2/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "da-marketplace-ui",
-  "version": "0.0.1",
+  "version": "0.1.8",
```
Comment (alexmatson-da) [inline ui2/package.json:3]: Bumped up the ui2 app version to get the `make package` target working properly for Daml Hub deployment again

File: ui2/src/context/ServicesContext.tsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ import {
   Service as TradingService,
   Request as TradingRequest,
 } from '@daml.js/da-marketplace/lib/Marketplace/Trading/Service/module';
-import { useStreamQueries } from '@daml/react';
+// import { useStreamQueries } from '@daml/react';
```
Comment (Sarahb-da) [inline ui2/src/context/ServicesContext.tsx:27]: missed one

File: ui2/src/pages/distribution/auction/Auctions.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { withRouter, RouteComponentProps } from 'react-router-dom';
-import { useStreamQueries } from '@daml/react';
+// import { useStreamQueries } from '@daml/react';
```
Comment (Sarahb-da) [inline ui2/src/pages/distribution/auction/Auctions.tsx:3]: here too

File: ui2/src/pages/distribution/bidding/Auctions.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { withRouter, RouteComponentProps } from 'react-router-dom';
-import { useStreamQueries } from '@daml/react';
+// import { useStreamQueries } from '@daml/react';
```
Comment (Sarahb-da) [inline ui2/src/pages/distribution/bidding/Auctions.tsx:3]: :)

File: ui2/src/pages/distribution/auction/Auctions.tsx (hunk 3 above)
Comment (Sarahb-da) [reply ui2/src/pages/distribution/auction/Auctions.tsx:3]: and a few others..

**Review by Sarahb-da — APPROVED** [review]
lgtm - a few lines still commented out in some places

**Review by alexgraham-da — APPROVED** [review]
looks good 👍

**Pull request merged**